src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -1754,7 +1755,8 @@ File 1
             verifier.VerifyDiagnostics();
         }
 
-        [Fact]
+        // Disabling to track down deadlock
```

Comment:
Do we have a bug tracking this?

Reply:
This is a mistake. Thanks for catching. Will undo.